docs/software/SDK/Baballonia Overview.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: Overview of Baballonia SDK
+sidebar_position: 1
+---
+
+# Overview of Baballonia SDK
+
+The Baballonia SDK provides developers with the tools and interfaces necessary to integrate eye-tracking and face-tracking capabilities into their applications. This SDK is designed to work seamlessly with the Baballonia software, allowing for easy access to processed tracking data and raw camera streams.
```

Comment:
While this statement isn't entirely false, it also isn't entirely true - the SDK exists to give developers the ability to add their own camera captures. 

Say, you had an analog video source and you wanted to feed in into Baballonia. You would use the SDK for this.